cmd/compose/ps.go

```diff
@@ -142,21 +144,18 @@ SERVICES:
 
 	return formatter.Print(containers, opts.Format, os.Stdout,
 		writer(containers),
-		"NAME", "COMMAND", "SERVICE", "STATUS", "PORTS")
+		"NAME", "IMAGE", "COMMAND", "SERVICE", "CREATED", "STATUS", "PORTS")
```

Comment:
Not sure. We would get this path from the current compose file, so this would only work when ran with a `--file` but not with `--project-name`, and even with this would assume the current compose file is in sync with listed containers.